generals/agents/expander_agent.py

```diff
@@ -49,12 +58,12 @@ def act(self, observation):
             action_index = np.random.choice(len(valid_actions))
             action = valid_actions[action_index]
 
-        pass_turn = 0  # 0 for not passing the turn, 1 for passing the turn
-        split_army = 0  # 0 for not splitting the army, 1 for splitting the army
-        cell = np.array([action[0], action[1]])
-        direction = action[2]
-
-        action = (pass_turn, cell, direction, split_army)
+        action = {
+            "pass": 0,
+            "cell": action[:2],
+            "direction": action[2],
+            "split": 0,
+        }
```

Comment:
Is there a way to make `action` from `valid_actions` to also have `"cell"` and `"direction"`?
Now there are weird manipulations like `action[:2]` or `action[:-1]` to extract relevant data.